CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Change Log
+
+## 1.0.0
+- Expose new configs in `values.yaml` to enable users to configure Fluentd to their liking. See the new `fluentd` section in `values.yaml` for more details.
+- Make several changes to `values.yaml` for consistency and understandability:
+
+Old Config | New Config
+|:-------- |:--------
+`sumologic.eventCollectionEnabled` | `fluentd.events.enabled`
+`sumologic.logFormat` | `fluentd.logs.output.logFormat`
+`sumologic.flushInterval` | `fluentd.buffer.flushInterval`
+`sumologic.numThreads` | `fluentd.buffer.numThreads`
+`sumologic.chunkLimitSize` | `fluentd.buffer.chunkLimitSize`
+`sumologic.totalLimitSize` | `fluentd.buffer.totalLimitSize`
+`sumologic.sourceName` | `fluentd.logs.containers.sourceName`
+`sumologic.sourceCategory` | `fluentd.logs.containers.sourceCategory`
+`sumologic.sourceCategoryPrefix` | `fluentd.logs.containers.sourceCategoryPrefix`
+`sumologic.sourceCategoryReplaceDash` | `fluentd.logs.containers.sourceCategoryReplaceDash`
+`sumologic.verifySsl` | `fluentd.verifySsl`
+`sumologic.excludeContainerRegex` | `fluentd.logs.containers.excludeContainerRegex`
+`sumologic.excludeHostRegex` | `fluentd.logs.containers.excludeHostRegex`
+`sumologic.excludeNamespaceRegex` | `fluentd.logs.containers.excludeNamespaceRegex`
+`sumologic.excludePodRegex` | `fluentd.logs.containers.excludePodRegex`
+`sumologic.fluentdLogLevel` | `fluentd.logLevel`
+`sumologic.watchResourceEventsOverrides` | `fluentd.events.watchResourceEventsOverrides`
+`sumologic.k8sMetadataFilter` | `fluentd.logs.containers.k8sMetadataFilter`
+
+- Remove some outdated configs in `values.yaml`
+  - sumologic.kubernetesMeta
+  - sumologic.kubernetesMetaReduce
+  > To preserve the behavior of `kubernetesMeta` or `kubernetesMetaReduce`, we recommend using the `fields` log_format, which by default strips this metadata from the body of the log message.
```

Comment:
kinda weird this shows up as grey-ed out text. Could these just be bullet points with `-` ?